plugins/trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/search_rules/action.py

```diff
@@ -47,7 +50,7 @@ def search_rule_by_cve(self, cve: str) -> Tuple[set, set, set]:
 
         # Check if matching IPS rules were found
         if response_data["intrusionPreventionRules"]:
-            hits = len(response_data["intrusionPreventionRules"])
+            # hits = len(response_data["intrusionPreventionRules"])
```

Comment:
Is this meant to be commented out? JW because I'm unsure this has anything to do with memory leaks

Reply:
not used in the code. Commented out because detected as unused by prospector, I think I will delete

Reply:
deleted, unusued code